changelog.d/9619.bugfix

```diff
@@ -0,0 +1 @@
+Prevent attempting to bundle aggregations for state events in /context APIs.
```

Comment:
This isn't really a bug? And if it was we should say what the effect was here.

I believe this is more about improving performance?

Reply:
Good point, I found it a bit ambiguous but I think `misc` does make a bit more sense.